src/utils/sivilstand.ts

```diff
@@ -1,43 +1,65 @@
 import { IntlShape } from 'react-intl'
 
-export function checkHarSamboer(sivilstand: Sivilstand): boolean {
-  return ['GIFT', 'REGISTRERT_PARTNER'].includes(sivilstand)
+export function checkHarSamboer(sivilstand?: UtvidetSivilstand): boolean {
+  if (!sivilstand) return false
+  return ['GIFT', 'REGISTRERT_PARTNER', 'SAMBOER'].includes(sivilstand)
+}
+
+export function checkUkjentSivilstand(sivilstand?: UtvidetSivilstand): boolean {
```

Comment:
Kan navngivning forbedres slik at den er mer beskrivende? For booleans er det f.eks nyttig med is* has* should* som prefixsynes jeg. F.eks `checkIsSivilstandUkjent` eller `isSivilstandUkjent`